services/codecov/codecov.service.js

```diff
@@ -46,63 +76,58 @@ module.exports = class Codecov extends LegacyService {
           repo: 'example-python',
           branch: 'master',
         },
-        staticPreview: { label: 'coverage', message: '90%', color: 'green' },
+        staticPreview: this.render({ coverage: 90 }),
       },
       {
-        title: 'Codecov private',
-        pattern: 'token/:token/:vcsName/:user/:repo',
+        title: 'Codecov private repository',
```

Comment:
This doesn't actually have to be a separate example anymore does it? I believe I can/should add the token query param to the two above examples instead

Reply:
Yup, that makes sense to me. Some of the other ones like this, like the npm custom registry examples, probably should be converted in the same way.